nextflow_schema.json

```diff
@@ -504,6 +504,55 @@
                     "description": "Specify the -k parameter for BWA mem i.e. the  minimum seed length",
                     "help_text": "Configures the re-seeding used in BWA-MEM. Default is set to BWA default.\n\n> Modifies BWA-MEM parameter: `-r`",
                     "fa_icon": "fas fa-angle-double-left"
+                },
+                "mapping_bowtie2_alignmode": {
+                    "type": "string",
+                    "default": "local",
+                    "description": "Specify the bowtie2 alignment mode. Options: 'local', 'end-to-end'.",
+                    "help_text": "The type of read alignment to use. Options are 'local' or 'end-to-end'. Local allows only partial alignment of read, with ends of reads possibly 'soft-clipped' (i.e. remain unaligned/ignored), if the soft-clipped alignment provides best alignment score. End-to-end requires all nucleotides to be aligned. \nDefault: 'local', following [Cahill et al (2018)](https://doi.org/10.1093/molbev/msy018) and [Poullet and Orlando 2020](https://www.frontiersin.org/articles/10.3389/fevo.2020.00105/full)",
+                    "fa_icon": "fas fa-arrows-alt-h"
+                },
+                "mapping_bowtie2_sensitivity": {
+                    "type": "string",
+                    "default": "sensitive",
+                    "description": "Specify the level of sensitivity for the bowtie2 alignment mode. Options: 'no-preset', 'very-fast', 'fast', 'sensitive', 'very-sensitive'.",
+                    "help_text": "The Bowtie2 'preset' to use. Options: 'no-preset' 'very-fast', 'fast', 'sensitive', or 'very-sensitive'. These strings apply to both --bt2_alignmode options. See the Bowtie2 manual for actual settings. \nDefault: 'sensitive' (following [Poullet and Orlando (2020)](https://www.frontiersin.org/articles/10.3389/fevo.2020.00105/full), when running damaged-data without UDG treatment)",
```

Comment:
Missing here (and for alignmode) the 'Modifies XYZ' section. Also `--bt2_alignmode` is the eager2 parameter right?